### DISCUSSION TITLE
The High-frequency telemetry data.

### DISCUSSION THREAD

**Discussion by fix-all** [eclipse-ditto/ditto#1898] [Q&A] [answered]
hi,
  What suggestions do you have for handling high-frequency telemetry data, such as vehicle location information reported at 10 messages per second? Would it be advisable to update to Thing.features?

**Comment by thjaeckle** [comment, marked as answer]
I would not use Ditto for such a use-case (high frequency telemetry ingestion).
I would update a twin in Ditto e.g. only once per minute - and ingest the high-frequency telemetry data into a Database suitable for that, like e.g. Apache IoTDB or a timeseries database.

**Reply by fix-all** [reply]
Thank you for your advice.
